config/default-disco.js

```diff
@@ -55,6 +55,7 @@ module.exports = {
   taarParamsToUse: [
```

Comment:
change to `discoParamsToUse`? 😉

Reply:
Good idea 👍 I'll do that as a follow-up patch.